### ISSUE TITLE
[TWW] Both zh client broke because localization error when start.

### ISSUE THREAD

**Issue by EKE00372** [Stanzilla/WoWUIBugs#577]
`Blizzard_FriendsFrame/Mainline/Localization.lua:13: attempt to index global 'WhoFrameDropDown' (a nil value)`

Happened on 11.0.0.54935 just update.

It borke both zhTW and zhCN client when start, make client stop with black screen.

```
local function LocalizeFriendsFrame_zh()
	ADDFRIENDFRAME_WOWHEIGHT = 232;
	ADDFRIENDFRAME_BNETHEIGHT = 310;
	AddFriendNameEditBox:SetPoint("TOP", 0, -144);

	for i=1, (FriendsFrame.numTabs or 0) do
		tabName = "FriendsFrameTab"..i;
		_G[tabName].Text:SetPoint("CENTER", tabName, "CENTER", 0, 5);
	end

	WhoFrameDropDown:SetPoint("TOPLEFT", WhoFrameColumnHeader2, "TOPLEFT", -15, 1);
end
```

**Comment by Meorawr** [comment]
This has already been fixed in an internal build; will keep it open and close it when it lands.

**Comment by Meorawr** [comment]
Fixed in 11.0.0.50000; also shouldn't entirely brick the UI going forward.
